src/CMakeLists.txt

```diff
@@ -493,6 +494,24 @@ add_subdirectory(include)
 add_subdirectory(librados)
 add_subdirectory(libradosstriper)
 
+if (WITH_MGR)
```

Comment:
better off adding another `CMakeLists.txt` in mgr, and moving this stuff into that file instead.